src/sbt-test/dumpLicenseReport/dep-exclusions-report/example.sbt

```diff
@@ -0,0 +1,21 @@
+name := "example"
+
+libraryDependencies += "com.fasterxml.jackson.core" % "jackson-databind" % "2.5.4"
+libraryDependencies += "junit"                      % "junit"            % "4.12"  % "test"
+
+excludeDependencies += "org.scala-lang"
+
+licenseDepExclusions := {
+  case sbtlicensereport.license.DepModuleInfo("com.fasterxml.jackson.core", _, _) => true
```

Comment:
Ideally I would like this to be `case DepModuleInfo` rather than `case sbtlicensereport.license.DepModuleInfo` but it doesn't seem to be working.